src/native/eventpipe/ep-json-file.c

```diff
@@ -124,14 +124,16 @@ ep_json_file_write_event_data (
 	ep_char8_t method_name [MAX_METHOD_NAME_LEN];
 
 	for (uint32_t i = 0; i < ep_stack_contents_get_length (stack_contents); ++i) {
-		ep_rt_method_desc_t *method = ep_stack_contents_get_method (stack_contents, i);
+		// ep_rt_method_desc_t *method = ep_stack_contents_get_method (stack_contents, i);
```

Comment:
I guess this is just temporary, but we can adjust ep_stack_contents_get_method to correctly get expected method on checked build.